atomkraft/test/trace.py

```diff
@@ -0,0 +1,57 @@
+from os import PathLike
+from atomkraft.utils.project import project_root
+import os.path
+from datetime import datetime
+import pytest
+from ..reactor.reactor import get_reactor
+
+
+trace_test_stub = """
+from modelator.pytest.decorators import itf
+
+pytest_plugins = "{0}"
+
+@itf("{1}", keypath="{2}")
+def test_trace():
+    print("Successfully executed trace {1}")
+"""
```

Comment:
:100: for this!

but best to make the variable name capitalize because it is a constant.